unit_tests/tests/test_fuel_math.cpp

```diff
@@ -10,22 +10,22 @@ TEST(FuelMath, getStandardAirCharge) {
 	CONFIG(specs.displacement) = 1.839f;
 	CONFIG(specs.cylindersCount) = 4;
 
-	EXPECT_FLOAT_EQ(0.553539f, getStandardAirCharge(PASS_ENGINE_PARAMETER_SIGNATURE));
+	EXPECT_FLOAT_EQ(0.5535934f, getStandardAirCharge(PASS_ENGINE_PARAMETER_SIGNATURE));
 
 	// LS 5.3 liter v8
 	CONFIG(specs.displacement) = 5.327f;
 	CONFIG(specs.cylindersCount) = 8;
 
-	EXPECT_FLOAT_EQ(0.80171353f, getStandardAirCharge(PASS_ENGINE_PARAMETER_SIGNATURE));
+	EXPECT_FLOAT_EQ(0.80179232f, getStandardAirCharge(PASS_ENGINE_PARAMETER_SIGNATURE));
 
 	// Chainsaw - single cylinder 32cc
 	CONFIG(specs.displacement) = 0.032f;
 	CONFIG(specs.cylindersCount) = 1;
-	EXPECT_FLOAT_EQ(0.038528003, getStandardAirCharge(PASS_ENGINE_PARAMETER_SIGNATURE));
+	EXPECT_FLOAT_EQ(0.038531788f, getStandardAirCharge(PASS_ENGINE_PARAMETER_SIGNATURE));
```

Comment:
these changes are because the order of operations for floating point numbers can change their value 🙂 (in this case, only 0.01% change)